load_test.py

```diff
@@ -214,6 +214,13 @@ def main(args):
 
         utils.write_output(config, results_list)
 
+    # Terminate queues immediately on ^C
+    except KeyboardInterrupt:
+        stop_q.cancel_join_thread()
+        dataset_q.cancel_join_thread()
+        if warmup_q:
+            warmup_q.cancel_join_thread()
```

Comment:
thanks @sjmonson , really useful hack :) 

could you try to move
```
        warmup = config.get("warmup")
        if not warmup:
warmup_q = None
```
outside of the `try`? will make it safer for the `exit_gracefully`

Reply:
Done